pkg/api/bzz.go

```diff
@@ -84,55 +79,35 @@ func (s *server) bzzDownloadHandler(w http.ResponseWriter, r *http.Request) {
 	}
 
 	// we are expecting manifest Mime type here
-	if ManifestContentType != metadata.MimeType {
+	m, err := manifest.NewManifestReference(
+		ctx,
```

Comment:
nitpicking but we usually don't have line feeds between parameters on a method call

Reply:
I opted for this (on several places) because I figured it would be better (easier to read and compare) longer lines.
There is no official requirement, however my thoughts were that once the line goes over 100 characters it might be smart to consider other options (personally I try to go to 80). I also find limited line easier for terminal (but this might be due to my configuration).
If we are to decide to push single-line only, that is fine by me, we can add it to `CODING.md`, and I will change these.